COLLABORATOR_GUIDE.md

```diff
@@ -378,6 +379,16 @@ multiple commits. Commit metadata and the reason for the revert should be
 appended. Commit message rules about line length and subsystem can be ignored.
 A Pull Request should be raised and approved like any other change.
 
+### Additions to the Cryptography and Security APIs
+
+Semver-minor commits that add or change cryptograpy/security APIs should be
+treated with extra care. Due to the potential impact, it is important that
+these APIs be constructed to reduce the potential for incorrect
+usage.
+
+These commits must have an approval from at least one member from the 
+[security working group](https://github.com/nodejs/security-wg).
```

Comment:
Like others, I am fine with the recommendation around taking increased care, but I'm not sure if the Security-WG is the correct venue. The @nodejs/cryto team already exists for coverage of the crypto and openssl related items, and the @nodejs/security team already handles Node.js core security related items. Perhaps an alternative here would be to require approval from at least one member of the Security WG *or* either @nodejs/crypto or @nodejs/security?

Reply:
I was under the impression that the Security WG and @nodejs/security were one and the same thing. Is this not the case?

Other than that, requiring approval from either the crypto *or* security team seems like a good idea.

Reply:
> I was under the impression that the Security WG and @nodejs/security were one and the same thing. Is this not the case?

@joepie91 Not really, there is @nodejs/security and @nodejs/security-wg.

Reply:
> I was under the impression that the Security WG and @nodejs/security were one and the same thing. Is this not the case?

They are not at all the same thing. I suggested not naming the WG "Security" to avoid exactly this kind of confusion. I don't know what I *would* name it though. Maybe "Security-and-Something-Else WG" or "Something-Else and Security WG".

Reply:
Oh, I guess I should explain the difference. Or at least provide links. `security-wg` is the Security Working Group consisting largely of interested parties but not necessarily (or even largely) people who do a lot of work themselves on Node.js core. More details are at https://github.com/nodejs/security-wg.

Meanwhile, `security` is the team briefly described at https://github.com/nodejs/security-wg/blob/master/processes/security_team_membership_policy.md. That team existed before `security-wg`.